source/agora/common/EnrollmentManager.d

```diff
@@ -0,0 +1,407 @@
+/*******************************************************************************
+
+    Contains supporting code for enrollment process.
+
+    Copyright:
+        Copyright (c) 2019 BOS Platform Foundation Korea
+        All rights reserved.
+
+    License:
+        MIT License. See LICENSE for details.
+
+*******************************************************************************/
+
+module agora.common.EnrollmentManager;
+
+import agora.common.crypto.ECC;
+import agora.common.crypto.Key;
+import agora.common.crypto.Schnorr : Pair;
+import agora.common.crypto.Schnorr : Signature;
+import agora.common.crypto.Schnorr : sign;
```

Comment:
Merge those together (actually I'd say, just loose the selective import and make it a regular import)